epub33/core/index.html

```diff
@@ -7850,144 +7850,182 @@ <h4>Content document dimensions</h4>
+					<code>always</code>. In addition to using the <code>rendition:flow</code> property, EPUB
+					creators MAY override this behavior through an appropriate style sheet declaration, if the
+					reading system supports such overrides.</p>
+
+				<p>EPUB creators MUST NOT declare the <code>rendition:flow</code> property more than once.</p>
+
+				<p>They also MUST NOT declare the property using the <a href="#attrdef-refines"><code>refines</code>
+					attribute</a>. Refer to <a href="#flow-overrides"></a> for setting the property for
+					individual EPUB content documents.</p>
+
+				<figure id="fig-flow-paginated-single">
+					<figcaption>Rendering of a reflowable EPUB publication with a single spine item, and with the
+							<code>rendition:flow</code> set to <code>paginated</code>.</figcaption>
+					<img src="images/example_rendering_paginated_single_spine.svg" width="600"
+						aria-details="flow-paginated-single-diagram"
+						alt="The continuous progression of paginated content produced for a single document." />
+				</figure>
+
+				<details id="flow-paginated-single-diagram" class="desc">
+					<summary>Image description</summary>
+					<p>Three column-like rectangles linked left-to-middle and middle-to-right with respective
+						arrows, with a text flowing from one rectangle to the next one. The text is sectioned with
+						headers figuring 'Chapter 1', '2', and '3'. The leftmost rectangle is enclosed in a
+						schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-flow-paginated-multiple">
+					<figcaption>Rendering of a reflowable EPUB publication with multiple spine items, and with the
+						<code>rendition:flow</code> set to <code>paginated</code>.</figcaption>
+					<img src="images/example_rendering_paginated_multiple_spine.svg" width="600"
+						aria-details="flow-paginated-multiple-diagram"
+						alt="The continuous progression of paginated content produced for each document with transitions to
+						new pages between documents." />
+				</figure>
+
+				<details id="flow-paginated-multiple-diagram" class="desc">
+					<summary>Image description</summary>
+					<p>Three column-like rectangles linked left-to-middle and middle-to-right with respective
+						arrows, with a text flowing from one rectangle to the next one. The text is sectioned with
+						headers figuring 'Chapter 1', '2'. The section with 'Chapter 2' starts at the top of the
+						rightmost rectangle, leaving an empty space at the bottom of the middle rectangle. The
+						leftmost rectangle is enclosed in a schematic view of a tablet.</p>
+				</details>
+
+				<figure id="fig-fxl-flow-paginated">
+					<figcaption>Rendering of a fixed layout EPUB publication with multiple spine items, and with the <code>rendition:flow</code> set to <code>paginated</code>.</figcaption>
+					<img src="images/example_rendering_FXL_paginated_multiple_spine.svg" width="600" 
```

Comment:
```suggestion
					<img src="images/example_rendering_FXL_paginated_multiple_spine.svg" width="900" 
```

To make the size of the figure consistent with the others.

Reply:
Resolving this and the other comments without further actions; the change of the images made all these remarks moot.